modules/@apostrophecms/schema/index.js

```diff
@@ -1072,6 +1073,9 @@ module.exports = {
           if (!forwardRelationship) {
             fail('idsStorage property does not match the idsStorage property of any relationship in the schema for ' + field.withType + '. Hint: you are taking advantage of a relationship already being edited in the schema for that type, your idsStorage must be the same to find the data there.');
           }
+          if (forwardRelationship.fieldsStorage) {
+            field.fieldsStorage = forwardRelationship.fieldsStorage;
```

Comment:
Less likely, but if someone chooses to find the reverse relationship via specifying idsStorage, it makes sense to infer the other bit.